packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/EditingContextControllerIntegrationTests.java

```diff
@@ -126,12 +154,37 @@ public void givenEditingContextIdWhenQueryIsPerformedThenItsActionsAreReturned()
         assertThat(hasNextPage).isFalse();
 
         String startCursor = JsonPath.read(result, "$.data.viewer.editingContext.actions.pageInfo.startCursor");
-        assertThat(startCursor).isNull();
+        assertThat(startCursor).isNotBlank();
 
         String endCursor = JsonPath.read(result, "$.data.viewer.editingContext.actions.pageInfo.endCursor");
-        assertThat(endCursor).isNull();
+        assertThat(endCursor).isNotBlank();
 
         int count = JsonPath.read(result, "$.data.viewer.editingContext.actions.pageInfo.count");
-        assertThat(count).isZero();
+        assertThat(count).isGreaterThan(0);
+    }
+
+    @Test
+    @DisplayName("Given a project, when an editing context action is invoked, then the editing context is modified")
+    @Sql(scripts = {"/scripts/initialize.sql"}, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+    @Sql(scripts = {"/scripts/cleanup.sql"}, executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, config = @SqlConfig(transactionMode = SqlConfig.TransactionMode.ISOLATED))
+    public void givenProjectWhenAnEditingContextActionIsInvokedThenTheEditingContextIsModified() {
+        var editingContextEventInput = new EditingContextEventInput(UUID.randomUUID(), TestIdentifiers.STUDIO_PROJECT.toString());
+        var flux = this.graphQLRequestor.subscribe(GET_EDITING_CONTEXT_EVENT_SUBSCRIPTION, editingContextEventInput);
+
+        var input = new InvokeEditingContextActionInput(UUID.randomUUID(), TestIdentifiers.STUDIO_PROJECT.toString(), StudioEditingContextActionProvider.EMPTY_DOMAIN_ID);
+        Runnable invokeEditingContextActionTask = () -> {
+            var result = this.graphQLRequestor.execute(INVOKE_EDITING_CONTEXT_ACTION, input);
+
+            TestTransaction.flagForCommit();
+            TestTransaction.end();
+
+            String typename = JsonPath.read(result, "$.data.invokeEditingContextAction.__typename");
+            assertThat(typename).isEqualTo(SuccessPayload.class.getSimpleName());
+        };
+
+        StepVerifier.create(flux)
+                .then(invokeEditingContextActionTask)
+                .thenCancel()
```

Comment:
Isn't this missing some "expectNextMatches"?

Reply:
No it's on purpose here since nothing will be sent on this subscription when we subscribed to it.